e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -469,9 +474,18 @@ def setup_workspaces(
         for e in exps:
             if e.workspaceId not in wids:
                 continue
-            bindings.post_KillExperiment(session, id=e.id)
+            bindings.delete_DeleteExperiment(session, experimentId=e.id)
+        time.sleep(0.5)
```

Comment:
what's up with the sleeping? is this waiting on the deletes to finish? if so there's no way this doesn't flake eventually haha.